### PR TITLE
PLANET-4216 Admin Add/edit EN form fixes for checkbox radio fieldtype

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
[JIRA 4216](https://jira.greenpeace.org/browse/PLANET-4216)

- Added updated JS and CSS files from old EN form block repo to new Gutenberg EN form repo.
- Fix edit checkbox field issue on admin edit EN form page
- Added radio field type in `P4 field type` dropdown option
- Code clean up

### CODE REVIEW

File: admin/css/admin_en.css [hunk 1]
```diff
@@ -29,11 +29,10 @@ div.shortcode-ui-edit-shortcake_enform > div:nth-child(n+9):nth-child(odd) {
   margin-right: 40%;
 }
 
-.shortcode-ui-attribute-content_title {
-  display: none;
-}
-
-.shortcode-ui-attribute-content_description {
+.shortcode-ui-attribute-content_title,
```
Comment (kirdia) [inline admin/css/admin_en.css]: @sagarsdeshmukh could you clean up this file? most styles are shortcake related (shortcode-ui-**) or related with the classic editor one way or another.

File: admin/js/en_ui.js [hunk 2]
```diff
@@ -173,10 +173,14 @@ jQuery(function ($) {
           const en_form_style = $('input[name=en_form_style]:checked').val();
```
Comment (kirdia) [inline admin/js/en_ui.js]: same here, this file includes functionality for shortcake block in the classic editor

File: admin/js/en_ui_heading_view.js [hunk 3]
```diff
@@ -11,10 +11,14 @@ const editAttributeHeadingEN = sui.views.editAttributeField.extend({
     const en_form_style = $('input[name=en_form_style]:checked').val();
```
Comment (kirdia) [inline admin/js/en_ui_heading_view.js]: same here, this file includes functionality for the shortcake block in the classic editor

**Comment by dpivo** [comment]
Code looks good. But it seems like it doesn't show the 'Form Preview' Anymore. Maybe a setup issue on my side.

**Comment by sagarsdeshmukh** [comment]
> Code looks good. But it seems like it doesn't show the 'Form Preview' Anymore. Maybe a setup issue on my side.

Indeed the form preview is broken, we have a ticket for this issue - https://jira.greenpeace.org/browse/PLANET-4274

**Pull request merged**